pkg/webui/console/containers/gateways-table/index.js

```diff
@@ -116,7 +116,7 @@ class GatewaysTable extends React.Component {
 
       return getGatewaysList(
         params,
-        ['name', 'description', 'frequency_plan_id', 'gateway_server_address'],
+        ['name', 'description', 'frequency_plan_ids', 'gateway_server_address'],
```

Comment:
I don't think this alone will be sufficient, since other parts of the console still use the deprecated `frequency_plan_id`.

Reply:
Well, this removes the warning described in the issue

Reply:
Sure, it may remove the warning, but if the table still expects `frequency_plan_id`, you're going to have an empty column.

Reply:
Yes, but the IS returns both `frequency_plan_id` and `frequency_plan_ids`. Currently, everything in the Console is designed around `frequency_plan_id` being a string. The idea of this PR is to remove the warning.

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/2993 already removed that warning on the server side, so I guess this PR is no longer needed.

Reply:
So what to do here? How come we both use `id` and `ids` in the first place?